deb/var/lib/polkit-1/localauthority/10-vendor.d/homebridge.pkla

```diff
@@ -0,0 +1,9 @@
+[Homebridge RegisterService]
+Identity=unix-user:homebridge
+Action=org.freedesktop.resolve1.register-service
+ResultAny=yes
+
+[Homebridge UnregisterService]
+Identity=unix-user:homebridge
```

Comment:
The systemd service user can be overridden and a different user selected:

https://github.com/homebridge/homebridge-apt-pkg#customising-the-systemd-service-file

Does this care if the service is running as the `homebridge` user? Or does the `homebridge` user just have to exist on the system?

Reply:
Good point. This needs to match the user that is used. What's the best way to do that?

Reply:
Oh, hm. The end user might customize it post-install. I'm not sure what the best way is to handle that.

Reply:
Scripts here run as `root` each time the service is started:

https://github.com/homebridge/homebridge-apt-pkg/tree/master/deb/etc/hb-service/homebridge/prestart.d

The `$USER` env var is conveniently still the service user.

We can probably add another script to check and update the `homebridge.pkla` file each time the service is started.

Reply:
One option, I suppose, would be to make the policy file use the `homebridge` group and then explicitly specify that in the service unit, so that even if the user is overridden then it's still okay. Lemme take a look at the best way to do that.

Reply:
Figured it out. I'll revise the PR momentarily.

Reply:
What scenarios what the user customise the `homebridge.pkla` file (apart from the user)? I'm not familiar with PolicyKit.

Reply:
Unfortunately, the only parameters available to us are `unix-user:USERNAME` and `unix-group:GROUPNAME`, but the latter should be sufficient for our purposes.

Reply:
The $USER may not be in a `homebridge` group.

You can probably use a pre-start script and `sed` to find the line starting with `unix-user:` and replace it with `unix-user:$USER`.

Reply:
The systemd directive `SupplementaryGroups=` can be used to specify a list of additional groups to add when executing the script. That's what I intend to use.